src/crud.py

```
@@ -1,7 +1,13 @@
 import os
 from collections.abc import Sequence
 from logging import getLogger
-from typing import Optional
+from sqlalchemy.sql.selectable import Select
+from sqlalchemy.sql.selectable import Select
```

Ruff (0.11.9)

Redefinition of unused `Select` from line 4

Remove definition: `Select`

(F811)